x/evm/keeper/migrate.go

```diff
@@ -49,6 +58,75 @@ func removeExternalTokenBurnerCode(ctx sdk.Context, ck chainKeeper) error {
 	return nil
 }
 
+func migrateCommandBatchSignature(ctx sdk.Context, ck chainKeeper, signer types.Signer, m types.MultisigKeeper) error {
+	commandBatchID := ck.getLatestSignedCommandBatchID(ctx)
+	for {
+		commandBatchMetadata := ck.getCommandBatchMetadata(ctx, commandBatchID)
+		if commandBatchMetadata.ID == nil {
+			break
+		}
+		commandBatchID = commandBatchMetadata.PrevBatchedCommandsID
+
+		sigID := hex.EncodeToString(commandBatchMetadata.ID)
+
+		if commandBatchMetadata.Status != types.BatchSigned {
```

Comment:
I wonder should we mark them as aborted if not `BatchSigned` since tss is gone and there's nothing going to mark it at timeout.